platform/kube/ingressstatus.go

```diff
@@ -57,11 +59,16 @@ func NewIngressStatusSyncer(mesh *config.ProxyMeshConfig, client *Client,
 		&v1beta1.Ingress{}, options.ResyncPeriod, cache.Indexers{},
 	)
 
+	var publishService string
+	if mesh.IngressService != "" {
```

Comment:
If this field isn't defined the status syncer will update the ingress resource with the node IP. This is somewhat useful if you are using a NodePort setup, as in Minikube. Though in that case you'll still need to use the 30000+ port.